build.sbt

```diff
@@ -79,7 +79,10 @@ lazy val semanticdb3 = crossProject
   .settings(
     publishableSettings,
     protobufSettings,
-    PB.protoSources.in(Compile) := Seq(file("semanticdb/semanticdb3")),
+    PB.protoSources.in(Compile) := Seq(
+      file("semanticdb/semanticdb2"),
+      file("semanticdb/semanticdb3")
+    ),
```

Comment:
Let's split `semanticdb2` into a separate project, e.g. by reverting https://github.com/scalameta/scalameta/commit/f973504e180fc5f59db3fc3267c4c00c827e5870. This way those who are interested just in `semanticdb3` won't have to download generated code for `semanticdb2`.